src/gui/menu_species.cpp

```diff
@@ -434,10 +437,15 @@ void DissolveWindow::on_SpeciesScaleChargesAction_triggered(bool checked)
     if (!species)
         return;
 
-    static ScaleChargesDialog dialog(this);
+    if (!modifyChargesModel)
+    {
+        modifyChargesModel = new ModifyChargesModel;
+    }
+
+    ModifyChargesDialog dialog(this, modifyChargesModel, ModifyChargesModel::Scaling);
```

Comment:
If the model is not required to persist between uses of `ModifyChargesDialog` I still wonder whether we're better just doing something like this and avoiding the class member variable altogether:
```suggestion
    ModifyChargesDialog dialog(this, new ModifyChargesModel(), ModifyChargesModel::Scaling);
```

Reply:
The model _should_ persist between uses of the dialog, so as the user switches between different instances of the dialog (scale, smooth, reduce sig fig) the established values for each do not simply go out of scope and are lost. These values must be contained in the model.